### PR TITLE
Add typing indicator animation

### PR DESCRIPTION
**Pull request by Vivihung**
### Motivation and Context
Today, the app did nothing after a user submitted a chat. The user doesn't know if the app is working or not. Therefore, we need a visual animation when the bot prepares a response. It could improve the user experience on the perceptional response time.

### Description
- Leverage the latest Fluent UI Animation component to render the loading balls animation with frames.
- Checkin the loading balls frames as an SVG
- Add TypeScript support in importing SVG

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [ ] ~The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`~
- [ ] ~All unit tests pass, and I have added new tests where possible~
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatRoom.tsx [hunk 1]
```diff
@@ -81,8 +85,15 @@ export const ChatRoom: React.FC = () => {
             sender: account?.homeAccountId,
             content: value,
         };
+        setIsBotTyping(true);
         dispatch(updateConversation({ message: chatInput }));
-        await chat.getResponse(value, selectedId);
+        try {
+            await chat.getResponse(value, selectedId);
+        } catch (error) {
+            setIsBotTyping(false);
```
Comment (SergeyMenshykh) [inline samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatRoom.tsx:93]: nit: the line #93 and #96 can go to finally block of the try statement and catch block can be removed.

File: samples/apps/copilot-chat-app/WebApp/src/components/chat/typing-indicator/TypingIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { Image, makeStyles } from '@fluentui/react-components';
+import React from 'react';
+import typingBalls from '../../../assets/typing-balls-light.svg';
+//const typingBalls = require('../../../assets/typing-balls-light.svg');
```
Comment (SergeyMenshykh) [inline samples/apps/copilot-chat-app/WebApp/src/components/chat/typing-indicator/TypingIndicator.tsx:6]: nit: dead code? :)

**Review by SergeyMenshykh — DISMISSED** [review]

File: samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatRoom.tsx (hunk 1 above)
Comment (Vivihung) [reply samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatRoom.tsx:93]: Good catch!

File: samples/apps/copilot-chat-app/WebApp/src/components/chat/typing-indicator/TypingIndicator.tsx (hunk 2 above)
Comment (Vivihung) [reply samples/apps/copilot-chat-app/WebApp/src/components/chat/typing-indicator/TypingIndicator.tsx:6]: yes... It is indeed a dead code. will remove it.

**Review by adrianwyatt — APPROVED** [review]

**Pull request merged**